test_framework/terraform/aws/centos/user-data-scripts/provision_rke.sh.tpl

```diff
@@ -2,13 +2,21 @@
 
 DOCKER_VERSION=20.10
 
+sed -i 's#^SELINUX=.*$#SELINUX='"${selinux_mode}"'#' /etc/selinux/config
+
+if [[ ${selinux_mode} == "enforcing" ]] ; then
+	setenforce  1
+elif [[  ${selinux_mode} == "permissive" ]]; then
+	setenforce  0
+fi
+
 sudo yum update -y
 sudo yum group install -y "Development Tools"
 sudo yum install -y iscsi-initiator-utils nfs-utils nfs4-acl-tools
 sudo systemctl -q enable iscsid
 sudo systemctl start iscsid
 
-until (curl https://releases.rancher.com/install-docker/${DOCKER_VERSION}.sh | sudo sh); do
+until (curl https://releases.rancher.com/install-docker/$${DOCKER_VERSION}.sh | sudo sh); do
```

Comment:
Typo `$${DOCKER_VERSION}.sh`

Reply:
This is not a typo, since the file is a template file, I've use use "$$" for variable interpolation whenever the variable is defined internally in the script not as Terraform variable.

Reply:
Got it, thanks